composer.json

```diff
@@ -23,6 +23,7 @@
     "friends-of-behat/suite-settings-extension": "^1.0",
     "friends-of-behat/symfony-extension": "^2.1",
     "friends-of-behat/variadic-extension": "^1.3",
+    "friendsofsymfony/oauth-server-bundle": ">2.0.0-alpha.0 ^2.0@dev",
```

Comment:
That probably should be `"friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev",`, at least that's how we are making it in most of the plugins

Reply:
Good point but not this time. This plugin is compatible with Sylius 1.9 and 1.10. It creates issues and does not pass tests otherwise. Check for example official Sylius InvoicingPlugin, they did the same.